radioapp/zeroconf.py

```diff
@@ -0,0 +1,20 @@
+from zeroconf import ServiceBrowser, Zeroconf
+import threading
+
+
+class LmrListener(object): #declare this before calling PairingScan
```

Comment:
This could probably be encapsulated so that the client code only needs to know about the methods `start`, `stop`, and `register_callback`.

Reply:
Not sure what you mean by those so I've implemented some functions in that direction. I don't recall any of those methods in the documentation anywhere.

Reply:
I believe he means creating a sort of wrapper function around the internal functions. So basically make add_service() private and then create a start(), a simpler function that takes less arguments. The type and name can be static members of the class, or constants. I am assuming that stuff won't change.

Can you confirm this for me zane. 


EDIT: Realized that there are start and stop functions. So I believe Zane you just want these private?